frontend/src/components/Card/Card.jsx

```diff
@@ -12,7 +12,7 @@ export default class Card extends Component {
   makeTitle(here) {
       if(here === true) {
           return (
-              <div>
+              <div className={styles.fitTitle}>
```

Comment:
Hmm, I see why you used `fitTitle`. I guess that's okay, but I have a feeling that `fitTitle` will still match `fit-title` -- same as how you can use `fontWeight` in JSX while it's `font-weight` in CSS. /shrug

Reply:
fair enough

Reply:
fair enough